.nycrc

```diff
@@ -0,0 +1,27 @@
+{
+  "cache": false,
+  "check-coverage": true,
+  "extension": [
+    ".ts"
+  ],
+  "include": [
+    "test/**/*.ts"
```

Comment:
coverage should be calculated on `src`, not `test`

Reply:
done